python/lsst/sims/catUtils/baseCatalogModels/OpSim3_61DBObject.py

```diff
@@ -107,15 +104,25 @@ def executeConstrainedQuery(self, spatialBound, constraint=None):
         results = self.execute_arbitrary(query, dtype=dtype)
         return results
 
-    def getObservationMetaData(self, obshistid, radiusDeg, makeCircBounds=True, makeBoxBounds=False):
+    def getObservationMetaData(self, searchCenter, searchRadius, constraint=None,
+                               fovRadius=1.75, makeCircBounds=True, makeBoxBounds=False):
         """
-        This class creates an ObservationMetaData object suitable for generating a PhoSim input
-        catalog based on an actual OpSim pointing.
+        This method will perform a query on the OpSim3_61 for pointings within a given region
```

Comment:
I'd actually rather not, for two reasons:

1) this is a very old OpSim run and I do not think its documentation exists anywhere

2) Because this is implemented as a `CatalogDBObject`, users can use the `get_column_names()` to inspect the available columns.